data/src/main/java/app/tivi/data/entities/Episode.kt

```diff
@@ -41,7 +41,7 @@ import org.threeten.bp.OffsetDateTime
     ]
 )
 data class Episode(
-    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "id")
```

Comment:
What's the thinking behind this change? R8 obfuscation *might* rename the field, which would then break the mapping.

Reply:
Room will keep fields in Entity classes, and uses the field names as column names by default.

Reply:
We can even remove all redundant `@ColumnInfo`s.

Reply:
I prefer the explicit `@ColumnInfo` as it makes everything very clear. Please revert this commit.

Reply:
Done.

Reply:
Thanks!